pwa/src/components/applications/applicationForm.tsx

```diff
@@ -16,6 +16,7 @@ import APIContext from "../../apiService/apiContext";
 import LoadingOverlay from "../loadingOverlay/loadingOverlay";
 import { HeaderContext } from "../../context/headerContext";
 import { AlertContext } from "../../context/alertContext";
+import ApplicationsTable from "./applicationsTable";
```

Comment:
I don't think we're actually using this import.

Reply:
check